casestudies/load-testing/README.md

```diff
@@ -0,0 +1,70 @@
+# ACK Performance at Scale
+
+In this usecase, we would deploy 10K SNS Topics, 20K SQS Queues and 20K SNS Subscriptions i.e 50K resources at a time using SQS and SNS ACK Controllers to examine performance of ACK and other metrics.
+
+**Testing environment:**
+* EKS cluster
+* ACK SNS and AWS SQS Controller
+* Prometheus + Grafana Stack
+
+![](ack.drawio.png)
+
+Prometheus is configured to scrape ACK Controller metrics which will help us analyze the performance and other metrics like resource counts, reconciliation time etc.
+*The 50K resources would be deployed using a Helm Chart.*
+
+After installing the Helm chart, following were the observations:
+
+* <u>**Analysing ACK Controller's resources behaviour**</u>
+The default installation of SQS and SNS ACK Controllers via Helm Chart configures deployment's CPU and Memory resource limits to 100m and 128Mi respectively. On deploying 50K resources, both SQS and SNS Controllers pods got CRASHED with OOM Killed error.
```

Comment:
Add a subtitle "Test 1: default values, CPU 100m and Memory 128Mi"

Reply:
Added necessary subtitles for each test cases